include/avif/avif.h

```diff
@@ -1160,6 +1160,11 @@ typedef struct avifDecoder
     // Can be useful to decode the gain map image only.
     avifBool ignoreColorAndAlpha;
 #endif
+
+    // This is true when avifDecoderParse() detects an animation track in the image. If this is true, the image can be decoded
+    // either as an animated image sequence or the primary image item can be retrieved as a still image by setting
+    // avifDecoderSetSource to the appropriate source.
+    avifBool animationTrackPresent;
```

Comment:
Nit: I wonder if we should use the term "sequence" or "image sequence" in the name of this field.

Reply:
Yeah good idea. Done.